pkg/components/pluggable/discovery.go

```diff
@@ -47,13 +47,16 @@ func removeExt(fileName string) string {
 }
 
 const (
-	SocketFolderEnvVar  = "DAPR_COMPONENTS_SOCKETS_FOLDER"
-	defaultSocketFolder = "/tmp/dapr-components-sockets"
+	SocketsFolderEnvVar  = "DAPR_COMPONENTS_SOCKETS_FOLDER"
+	defaultSocketsFolder = "dapr-components-sockets"
```

Comment:
```suggestion
	DefaultSockets\Folder = "dapr-components-sockets"
```

Reply:
Wasn't sure if there was a reason for the existing naming, so didn't change it originally, but can do so.